sat/cli/status/status_module.py

```diff
@@ -1,7 +1,7 @@
 #
 # MIT License
 #
-# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+# (C) Copyright 2024 Hewlett Packard Enterprise Development LP
```

Comment:
The guidance for copyright years is to add the current year each time you make a change to a file rather than replace the old year. If it forms a contiguous range of years, they can be separated with a dash. Otherwise, they can be separated by a comma. So in this case:

```suggestion
# (C) Copyright 2022, 2024 Hewlett Packard Enterprise Development LP
```